auth-web/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth-web",
-  "version": "1.0.19",
+  "version": "1.1.0",
```

Comment:
Since the sbc-common-components is a major version jump (even though its changes are minimal), and since "19" is a pretty large patch number, I increased the  minor version here.

Reply:
Yeah I didn't want to bump sbc-common-components a major version. Faiz wanted to be consistent with other projects I believe.